receiver/riakreceiver/testdata/integration/Dockerfile.riak

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:20.04 as download
+FROM ubuntu:22.04 as download
```

Comment:
This upgrade causes the container to fail building

Reply:
Please see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21741